fixtures/case_entity.yaml

```diff
@@ -395,12 +395,17 @@ App\Entity\Embeddable\Address:
     city: Randers C
     postalCode: 8900
 
+App\Entity\Embeddable\Identification:
+  test-case-aarhus-1-identification:
+    type: CPR
+    identifier: 1234567890
+
+
```

Comment:
Remove this line (we usually only have one blank line between sections).

Reply:
Removed.